tests/NPSP.py

```diff
@@ -749,9 +755,11 @@ def click_object_manager_button(self,title):
 
     def page_scroll_to_locator(self, path, *args, **kwargs):
         locator = self.get_npsp_locator(path, *args, **kwargs)
+        import sys; sys.stdout=sys.__stdout__
+        import pdb; pdb.set_trace()
```

Comment:
stray pdb

Reply:
d'oh! Thanks for catching that.

Reply:
I've removed them and pushed the changes